lib/vrl/web-playground/public/index.html

```diff
@@ -88,83 +88,150 @@ <h1>[draft] VRL playground wasm</h1>
 
 
 .timestamp = to_unix_timestamp(ts)`,
-        language: 'python',
-        theme: 'vs-light',
-        minimap: { enabled: false }
-      });
-
-      window.eventEditor = monaco.editor.create(document.getElementById('container-event'), {
-        value: `{
+          language: "coffeescript",
+          theme: "vs-light",
+          // minimap: { enabled: false },
+          automaticLayout: true,
+        });
+
+        window.eventEditor = monaco.editor.create(document.getElementById("container-event"), {
+          value: `{
   "message": "<102>1 2020-12-22T15:22:31.111Z vector-user.biz su 2666 ID389 - Something went wrong"
 }`,
```

Comment:
With JSONL supported now, I wonder if it makes sense to make this initial example JSONL?